example/app/config.yml

```diff
@@ -423,6 +427,18 @@ jl:
     shared-2:
       name: rvl-internal
       type: shared
+  ecs:
+     ssh: true
+     keyPairName: jl
+  ems:
+    provider: CUMULUS
+    host: s-51625e6606b046448.server.transfer.us-east-1.amazonaws.com
```

Comment:
What is this pointing to now? Seems like this should be specified in the `.env` file?

Reply:
It's my personal SFTP server.  EMS server is fs1.ems.eosdis.nasa.gov.  I don't feel it needs to be in .env.